examples/live/bybit/bybit_request_custom_endpoint.py

```diff
@@ -0,0 +1,148 @@
+#  https://nautechsystems.io
+#
+#  Licensed under the GNU Lesser General Public License Version 3.0 (the "License");
+#  You may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at https://www.gnu.org/licenses/lgpl-3.0.en.html
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+# -------------------------------------------------------------------------------------------------
+
+import os
+from datetime import timedelta
+
+from nautilus_trader.adapters.bybit.common.enums import BybitInstrumentType
+from nautilus_trader.adapters.bybit.config import BybitDataClientConfig
+from nautilus_trader.adapters.bybit.config import BybitExecClientConfig
+from nautilus_trader.adapters.bybit.factories import BybitLiveDataClientFactory
+from nautilus_trader.adapters.bybit.factories import BybitLiveExecClientFactory
+from nautilus_trader.common import Environment
+from nautilus_trader.common.clock import TimeEvent
+from nautilus_trader.config import InstrumentProviderConfig
+from nautilus_trader.config import LiveExecEngineConfig
+from nautilus_trader.config import LoggingConfig
+from nautilus_trader.config import StrategyConfig
+from nautilus_trader.config import TradingNodeConfig
+from nautilus_trader.core.message import Request
+from nautilus_trader.core.nautilus_pyo3 import InstrumentId
+from nautilus_trader.core.uuid import UUID4
+from nautilus_trader.data.messages import DataResponse
+from nautilus_trader.live.node import TradingNode
+from nautilus_trader.trading import Strategy
+
+
+# *** THIS IS A TEST STRATEGY WITH NO ALPHA ADVANTAGE WHATSOEVER. ***
+# *** IT IS NOT INTENDED TO BE USED TO TRADE LIVE WITH REAL MONEY. ***
+
+# *** THIS INTEGRATION IS STILL UNDER CONSTRUCTION. ***
+# *** CONSIDER IT TO BE IN AN UNSTABLE BETA PHASE AND EXERCISE CAUTION. ***
+
+
+class RequestDemoStrategyConfig(StrategyConfig, frozen=True):
+    instrument_id: str
+    interval: int
+
+
+class RequestDemoStrategy(Strategy):
+    """
+    Strategy showcases how to request data from a custom bybit endpoint. Some adapter
+    endpoints are not connected to data engine, so they cannot be queried. We can use
+    the message bus to request data from these endpoints, as they are registered in
+    appropriate clients.
+
+    Also this strategy demonstrate:
+    - how to use the message bus to request data from a custom endpoint.
+    - how to use clock to schedule this request periodically.
+
+    """
+
+    def __init__(self, config: RequestDemoStrategyConfig):
+        super().__init__()
+        self.interval = config.interval
+        self.instrument_id = InstrumentId.from_str(config.instrument_id)
+        self.linear_request_ticker_uuid = UUID4()
```

Comment:
Each request should have a unique `UUID4`, it could cause downstream issues to be hanging onto the `request_id` reference and potentially reusing it with several in-flight

Reply:
Delete this, this demo strategy is not much simpler it uses `request_data` and listens on `on_historical_method` callback